### DISCUSSION TITLE
Oracle and MSSQL External Database for K3S

### DISCUSSION THREAD

**Discussion by zalseryani** [k3s-io/k3s#7086] [Q&A]
**Feature Request**
- Currently we are using K3S with External Postgresql Database or embedded ETCD DB. 

- It is just a feature reqeust, and we want to know if there are any plans to support k3s with oracle/mssql as external database.

Thank you.

**Comment by brandond** [comment]
We're not planning on adding support for any non-open-source datastores. The value proposition for that does seem to be very good, compared to any of the open-source SQL datastores, or etcd, all of which can be operated without licence fees.
